.run/bsh-ui.sh

```
@@ -0,0 +1,65 @@
+if [ -d "../${REPO_NAME}" ]; then
+  printf  "${COLOUR}Updating the local repository... ${LINE_END}"
+  cd ../${REPO_NAME}
+  git reset --hard
+  git checkout main
+  output=(git pull)
+
+  if echo "$output" | grep -q "Already up to date."; then
+    rebuild=false
+  fi
+else
+  printf  "${COLOUR}The repository does not exist. cloning the repository...${LINE_END}"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)